fhir-server/liberty-config/config/default/fhir-server-config-db2.json

```diff
@@ -9,6 +9,56 @@
             "checkReferenceTypes": true,
             "conditionalDeleteMaxNumber": 10
         },
+        "notifications": {
+            "common": {
+                "__comment_includeResourceTypes": [
+                    "QuestionnaireResponse",
+                    "CarePlan",
+                    "MedicationAdministration",
+                    "Device",
+                    "DeviceComponent",
+                    "DeviceMetric",
+                    "MedicationOrder",
+                    "Observation"
+                ]
+            },
+            "websocket": {
+                "enabled": true
+            },
+            "kafka": {
+                "enabled": false,
+                "topicName": "fhirNotifications",
+                "connectionProperties": {
+                    "group.id": "securing-kafka-group",
+                    "bootstrap.servers": "localhost:9093",
+                    "security.protocol": "SSL",
+                    "ssl.truststore.location": "resources/security/kafka.client.truststore.p12",
+                    "ssl.truststore.password": "change-password",
+                    "ssl.keystore.location": "resources/security/kafka.client.keystore.p12",
+                    "ssl.keystore.password": "change-password",
+                    "ssl.key.password": "change-password",
+                    "ssl.truststore.type": "PKCS12",
+                    "ssl.keystore.type": "PKCS12",
+                    "acks": "all",
+                    "retries": "60",
+                    "request.timeout.ms": "10000",
+                    "max.block.ms": "60000",
+                    "max.in.flight.requests.per.connection": "5"
+                }
+            },
+            "nats": {
+                "enabled": true,
```

Comment:
do we want to default to this for all? or selectively change it for the build?

Reply:
`fhir-server-config-db2.json` is the config used in our CI, so thats where I enabled it.  originally i had introduced a separate `fhir-server-config-nats.json` (and was gonna have a separate job for it in github actions), but I decided just to combine it with fhir-server-config-db2.json instead.

Reply:
`fhir-server-config.json` still has nats disabled by default